### PR TITLE
Add spark-processing bundle

### PR DESCRIPTION
**Pull request by arosales**
Test plan to add spark processing bundle

### CODE REVIEW

File: test_plans/spark-processing [hunk 1]
```diff
@@ -0,0 +1,21 @@
+bundle: bundle:spark-processing
+benchmark:
+    spark:
+        logisticregression
+    spark:
```
Comment (seman) [inline test_plans/spark-processing]: Multiple benchmarks are not supported

File: test_plans/spark-processing [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        streaming
+    spark:
+        svdplusplus
+    spark:
+        svm
+    spark:
+        trianglecount
+bundle_name: spark-processing
+bundle_file: bundle.yaml
+url: https://jujucharms.com/spark-processing/
```
Comment (seman) [inline test_plans/spark-processing]: This doesn't break anything. In near future, would like to add a link back to the charm store.

File: test_plans/spark-processing [hunk 3]
```diff
@@ -0,0 +1,21 @@
+bundle: bundle:spark-processing
```
Comment (seman) [inline test_plans/spark-processing]: Filename needs to ends with ".yaml"

File: test_plans/spark-processing (hunk 1 above)
Comment (marcoceppi) [inline test_plans/spark-processing]: Can we add multiple benchmarks? A lot of solutions include more than one benchmark action

**Comment by arosales** [comment]
@seman 
Thanks for the review I updated my branch per your suggestions. Specifically to add the yaml suffix (thanks for catching that), and removing all but one benchmark.

**Pull request merged**